src/ResourceManager/Insights/Commands.Insights/Metrics/NewAzureRmMetricFilterCommand.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System.Linq;
+using System.Management.Automation;
+using System.Text;
+
+namespace Microsoft.Azure.Commands.Insights.Metrics
+{
+    /// <summary>
+    /// Create a metric dimension filter
+    /// </summary>
+    [Cmdlet(VerbsCommon.New, "AzureRmMetricFilter"), OutputType(typeof(string))]
+    public class NewAzureRmMetricFilterCommand : MonitorCmdletBase
+    {
+        /// <summary>
+        /// Gets or sets the Dimension
+        /// </summary>
+        [Parameter(Position = 0, Mandatory = true, ValueFromPipelineByPropertyName = true, HelpMessage = "The dimension name")]
+        [ValidateNotNullOrEmpty]
+        public string Dimension { get; set; }
+
+        /// <summary>
+        /// Gets or sets the Operator
+        /// </summary>
+        [Parameter(Position = 1, Mandatory = true, ValueFromPipelineByPropertyName = true, HelpMessage = "The operator")]
+        [ValidateNotNullOrEmpty]
+        public string Operator { get; set; }
+
+        /// <summary>
+        /// Gets or sets the values list of the dimension. A comma-separated list of values for the dimension.
+        /// </summary>
+        [Parameter(Position = 2, Mandatory = true, ValueFromPipelineByPropertyName = true, HelpMessage = "The list of values for the dimension")]
+        [ValidateNotNullOrEmpty]
+        public string[] Values { get; set; }
```

Comment:
@pkamath13 @gucalder this should be renamed to `Value` to be conformant to PowerShell's singular noun rule for parameters